src/inbox/mod_inbox_utils.erl

```diff
@@ -135,3 +136,14 @@ reset_marker_to_bin(displayed) -> <<"displayed">>;
 reset_marker_to_bin(acknowledged) -> <<"acknowledged">>;
 reset_marker_to_bin(received) -> <<"received">>;
 reset_marker_to_bin(Unknown) -> throw({unknown_marker, Unknown}).
+
+-spec fill_namespace_attr(Msg :: exml:element()) -> exml:element().
+fill_namespace_attr(Msg = #xmlel{name = Name, attrs = Attrs}) when Name =:= <<"message">>  ->
```

Comment:
Matching on `<<"message">>` inside `#xmlel` would shorten this line significantly and make it much more readable :)